### ISSUE TITLE
NTR: atrioventricular valve regurgitation

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#7633]
**Preferred term label:**
atrioventricular valve regurgitation

**Synonyms**
atrioventricular valve insufficiency
AVSD AV valvar regurgitation

**Definition (free text, please give PubMed ID)**
The backward reflux of blood through the atrioventricular valve, due to insufficiency caused by disease, aging or congenital malformation.
MP:0006046

**Parent term (use hpo.jax.org/app)**
HP:0031650 'Abnormal atrioventricular valve physiology'

Needed for INCLUDE project

**Comment by pnrobinson** [comment]
In normal hearts, there is no single "atrioventricular valve"  -- are mitral and tricuspid valves. What would the use case be for an additional term?

**Comment by lopierra** [comment]
The term "atrioventricular valve" is used several times in our dataset - I think it refers to either mitral or tricuspid without specifying.

There is a term "atrioventricular valve regurgitation"  (MP:0006046) in MPO that had child terms "mitral valve regurgitation" and "triscuspid valve regurgitation". Could HPO have the same thing?

**Comment by pnrobinson** [comment]
@lopierra 
I think in human medicine the phrase can be used differently -- https://pubmed.ncbi.nlm.nih.gov/32419552/
In this case, "Atrioventricular Valve" is the valve present in congenital heart defects in which only a single ventricle is present.
Here it is used to mean mitral or tricuspid -- https://heart.bmj.com/content/107/12/947
I will add a term called 'atrioventricular valve regurgitation' as a grouping term and also a term 
Single atrioventricular valve regurgitation to refer to the congenital defect.

**Comment by pnrobinson** [comment]
added as HP:0034376
